static/restaurant/css/style.css

```diff
@@ -1,619 +1,989 @@
-    color: #56a621;
+.left__menu__lists ul li.active a {
+  color: #56a621;
 }
 
-
-.left__menu__lists ul li a{
-    color: #605f60;
-    font-weight: 600;
-    font-size: 16px;
-    width: 100%;
-    display: flex;
-    align-items: center;
-    padding: 1rem 0;
+.left__menu__lists ul li a {
+  color: #605f60;
+  font-weight: 600;
+  font-size: 16px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 1rem 0;
 }
 
-.left__menu__lists ul li a .fa{
-    margin: 0 20px 0 0;
-    width: 20px;
+.left__menu__lists ul li a .fa {
+  margin: 0 20px 0 0;
+  width: 20px;
 }
 
 .right__body__sec {
-    width: 100%;
+  width: 100%;
 }
 
 .rh-bd-tp-m-hl {
-    background: #faf9fa;
+  background: #faf9fa;
 }
 
-.tp-m ul li{
-    display: inline-flex;
-    margin: 0 30px 0 0;
+.tp-m ul li {
+  display: inline-flex;
+  margin: 0 30px 0 0;
 }
 
-.tp-m ul li a{
-    color: #a7a5a5;
-    font-size: 13px;
-    font-weight: 600;
-    padding: 15px 0;
+.tp-m ul li a {
+  color: #a7a5a5;
+  font-size: 13px;
+  font-weight: 600;
+  padding: 15px 0;
 }
 
-
-.tp-m ul li.active a{
-    color: #000;
-    border-bottom: 3px solid #56a621;
+.tp-m ul li.active a {
+  color: #000;
+  border-bottom: 3px solid #56a621;
 }
 
 .tp-m {
-    display: flex;
-    justify-content: space-between;
+  display: flex;
+  justify-content: space-between;
 }
 
-.fr-pr ul li a{
-    color: #56a621;
+.fr-pr ul li a {
+  color: #56a621;
 }
 
-.fr-pr ul li{
-    margin: 0;
+.fr-pr ul li {
+  margin: 0;
 }
 
-.fr-pr ul li a .fa{
-    margin: 0 10px 0 0;
+.fr-pr ul li a .fa {
+  margin: 0 10px 0 0;
 }
 
-.it-ac-hd{
-    border-bottom: 1px solid #ccc;
-    padding: 0 0 0.5rem 0;
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
+.it-ac-hd {
+  border-bottom: 1px solid #ccc;
+  padding: 0 0 0.5rem 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
 }
 
-.it-ac-hd .sc-hd h1{
-    margin: 0;
+.it-ac-hd .sc-hd h1 {
+  margin: 0;
 }
 
-
-.ad-fd-fm .fm-ls{
-    width: 70%;
+.ad-fd-fm .fm-ls {
+  width: 70%;
 }
 
-.up-im{
-    display: inline-flex;
-    position: relative;
+.up-im {
+  display: inline-flex;
+  position: relative;
 }
 
-.up-im input{
-    position: absolute;
-    width: 100% !important;
-    height: 100% !important;
-    opacity: 0;
+.up-im input {
+  position: absolute;
+  width: 100% !important;
+  height: 100% !important;
+  opacity: 0;
 }
 
-
-
-.up-im-bt{
-    width: 270px;
-    height: 220px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    text-align: center;
-    background: #fff;
-    border: 2px dashed #d2d2d2;
+.up-im-bt {
+  width: 270px;
+  height: 220px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  background: #fff;
+  border: 2px dashed #d2d2d2;
 }
 
-
-.up-im-cn{
-
+.up-im-cn {
 }
 
-
-
 .pv-im-hl {
-    width: 270px;
-    height: 220px;
-    margin: 0 0 0 2rem;
-}
-
-.pv-im-hl img{
```

Comment:
Add these things in the current css file too. We only need to copy and paste the css code that are mentioned below the comment that refers to the date September 7. You can see that comment in css file.